src/pydrex/visualisation.py

```diff
@@ -274,3 +281,192 @@ def corner_flow_2d(
         ax_mean.legend()
 
     _savefig_deep(fig, savefile)
+
+
+def set_polefig_axis(ax, ref_axes="xz"):
+    # NOTE: We could subclass matplotlib's Axes like Joe does in mplstereonet,
+    # but this turns out to be a lot of effort for not much gain...
+    ax.set_axis_off()
+    ax.set_xlim((-1.1, 1.1))
+    ax.set_ylim((-1.1, 1.1))
+    ax.set_aspect("equal")
+    _circle_points = np.linspace(0, np.pi * 2, 100)
+    ax.plot(np.cos(_circle_points), np.sin(_circle_points), linewidth=1, color="k")
+    ax.axhline(0, color="k", alpha=0.5)
+    ax.text(1.05, 0.5, ref_axes[0], verticalalignment="center", transform=ax.transAxes)
+    ax.axvline(0, color="k", alpha=0.5)
+    ax.text(
+        0.5, 1.05, ref_axes[1], horizontalalignment="center", transform=ax.transAxes
+    )
+
+
+def poles(orientations, ref_axes="xz", hkl=[1, 0, 0]):
```

Comment:
Just wondering is that is something `pyproj` can help you with?

Reply:
I looked into it but that requires an unnecessary conversion to spherical coords. Added some docs and comments though in the incoming commit.